cql3/restrictions/multi_column_restriction.hh

```diff
@@ -190,7 +190,7 @@ public:
     {
         using namespace expr;
         expression = binary_operator{
-            std::vector<column_value>(_column_defs.cbegin(), _column_defs.cend()), oper_t::EQ, _value};
+            expr::column_value_tuple{std::vector<column_value>(_column_defs.cbegin(), _column_defs.cend())}, oper_t::EQ, _value};
```

Comment:
Yuck!  There has to be a shorter way to express the simple idea of "tuple from these defs".  How about adding a constructor?

Reply:
Added a constructor.